bindings/python/pyproject.toml

```diff
@@ -12,6 +12,30 @@ classifiers = [
     "Programming Language :: Python :: Implementation :: CPython",
     "Programming Language :: Python :: Implementation :: PyPy",
 ]
+dependencies = [
+  "maturin==1.0.1",
+  "ruff==0.0.275",
+  "mypy==1.3.0",
+  "typos==1.15",
+  "pyarrow",
+]
```

Comment:
None of these should be required. I'd expect pyarrow to be in optional dependencies.